src/systems/systemItemsTable.component.test.tsx

```diff
@@ -63,6 +63,9 @@ describe('SystemItemsTable', () => {
     // Expand a group so all columns are rendered to improve test coverage
     // (expanding all causes an infinite loop due to an issue with details panels)
     await user.click(screen.getAllByRole('button', { name: 'Expand' })[0]);
+    //also unhide created column
+    await user.click(screen.getByRole('button', { name: 'Show/Hide columns' }));
+    await user.click(screen.getByText('Created'));
```

Comment:
I can see several warnings in the unit test logs, e.g. `Warning: An update to ForwardRef(FormControl) inside a test was not wrapped in act`. Some times these do just need to be wrapped, but usually as here I think replacing the getByRole with `await findByRole` will fix it as it also uses act but its only required when there are changes that need to be awaited.